nvtabular/inference/triton/model.py

```diff
@@ -56,6 +69,10 @@ def initialize(self, args):
         self.model_config = json.loads(args["model_config"])
         self.output_model = self.model_config["parameters"]["output_model"]["string_value"]
 
+        # recurse over all column groups, initializing operators for inference pipeline
+        initialized_column_groups = set()
+        self._initialize_ops(self.workflow.column_group, initialized_column_groups)
```

Comment:
The potential default value change mentioned above would make this:
```suggestion
        self._initialize_ops(self.workflow.column_group)
```